### PR TITLE
[INV-69] Configure CORS

### PR DESCRIPTION
**Pull request by Aziza-Azizova**
## Description
- Config stored in the ENV file
- Validation for ENV config updated
- CORS works

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -19,3 +19,6 @@ COOKIES_REFRESH_MS=
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 GOOGLE_CALLBACK_URL=http://localhost:3000/auth/google/redirect
+
+CORS_ENABLED=
+CORS_ORIGINS=
```
Comment (ShahobiddinIsmoilov) [inline .env.example:24]: You could also add CORS_ALLOWED_METHODS and CORS_ALLOWED_HEADERS

File: src/main.ts [hunk 2]
```diff
@@ -15,6 +15,17 @@ async function bootstrap() {
...
   });
 
+  const isCorsEnabled = configService.get<boolean>('CORS_ENABLED');
+  const corsOrigins = configService.get<string>('CORS_ORIGINS')?.split(',');
+
+  if (isCorsEnabled) {
+    app.enableCors({
+      origin: corsOrigins,
+      methods: 'GET, PUT, PATCH, POST, DELETE',
+      allowedHeaders: 'Content-Type, Authorization',
```
Comment (ShahobiddinIsmoilov) [inline src/main.ts:25]: We could take these values from env too

File: .env.example (hunk 1 above)
Comment (ShahobiddinIsmoilov) [inline .env.example:24]: Also, could you please write it like CORS_ALLOWED_ORIGINS for consistency

File: src/main.ts (hunk 2 above)
Comment (NegatiV-64) [reply src/main.ts:25]: I don't think that we need to limit headers

Comment (ShahobiddinIsmoilov) [reply src/main.ts:25]: Oh, okay

**Review by NegatiV-64 — APPROVED** [review]

**Pull request merged**